### ISSUE TITLE
3D Different Lightning Rendering OSX != Windows output

### ISSUE THREAD

**Issue by ruffiely** [godotengine/godot#28605]
**Godot version:**
3.1.1

**OS/device including version:**
OSX Macbook Version 10.14.4 
Intel Iris Plus Graphics 640

**Issue description:**
Actually I updated my project from 3.0.6 stable to 3.1.1 stable. Everything worked fine! The issue I encounter now is that my rendering scene is looking different on OSX and Windows and this shouldn't be the case by same configuration / objects etc. The Windows one is how it should look like

Windows look: https://ibb.co/S362T7C
OSX look: https://ibb.co/t2C5qc0

It seems like it is missing some light information or something. I deleted the .import folder and reimported everything still same issue. Maybe some one has a clue?

**Comment by volzhs** [comment]
_updated comment to see and compare screenshots easily_

**Comment by volzhs** [comment]
it looks like a directional light is off on osx screenshot.

**Comment by Calinou** [comment]
Which renderer is your project using (GLES2 or GLES3)?

**Comment by akien-mga** [comment]
Can you provide a minimal project that reproduces the issue?

**Comment by ruffiely** [comment]
Yeah so renderer GLES3 is used. The directional light is enabled, but it behaves completely weird as you can see in this video: https://youtu.be/wGITL2QCCXk
This is only on OSX on Windows it behaves like you would expect it. I will look if I can reproduce the problem in smaller context to share with you

**Comment by ruffiely** [comment]
Please checkout this repository with test project for the miss behaviour https://github.com/ruffiely/godot_directional_light_issue_osx

**Comment by KoBeWi** [comment]
Can anyone still reproduce this bug in Godot 3.2.3 or any later release?

**Comment by RuthgerD** [comment]
We have a similar [issue](https://github.com/ItJustWorksTM/smce-gd/issues/40) in our application on both 3.2.4 and even the 3.3 rc6. I cannot provide much more information as I dont have access to macos myself
macos:
![](https://user-images.githubusercontent.com/57626200/112620425-cfbc6b00-8e28-11eb-96a7-1f67bf7b3c9e.png)
linux:
![](https://cdn.discordapp.com/attachments/823998556553478226/824960081921704016/unknown.png)
